### DISCUSSION TITLE
求求求求求支持多线程渲染

### DISCUSSION THREAD

**Discussion by LinCTang** [Class-Widgets/Class-Widgets#307] [Feature]
### 检查清单

- [X] 我已更新到最新版 ![最新的版本号](https://img.shields.io/github/v/release/RinLit-233-shiroko/Class-Widgets?include_releases&style=flat-square&label=) ，并看过[最新提交](https://github.com/RinLit-233-shiroko/Class-Widgets/commits/main/)，确认我想要的功能还没有实现。
- [X] 我已在 [Issues](https://github.com/RinLit-233-shiroko/Class-Widgets/issues?q=label%3ABug) 中检索，确认这一 Bug 未被提交过。
- [X] 我认为我想要的功能是用户广泛需要的，插件不能替代的，且没有超出 **软件本来** 的开发目标，而非 **添加与课表及教学辅助无关的内容** 。
- [X] 我已知晓由于开发者时间精力有限，我提交的功能请求可能需要比较久的时间才能实现。

### 背景与动机

该男子班上电脑为8核16线程的低压cpu，单核性能非常弱，在使用时巨卡，只有大约4帧，任务管理器显示单核吃满

### 想要实现或优化的功能

实现多线程渲染

### 最后一步

- [X] 我认为上述的描述已经足以详细，以便开发人员理解您想要的功能。如果我的 issue 没有按照上述的要求填写，可能会被无条件关闭。

**Comment by baiyao105** [comment]
qt是单线程ui渲染模型,并行渲染并不太会在计划中
